Derulo/JSONPersistenceManager.swift

```diff
@@ -115,14 +90,13 @@ private struct JSONFileManager {
     static func write(jsonArray: [JSON], toFilename: String) {
         write(validJSON: jsonArray, toFilename: toFilename)
     }
-    
-    
+
     //MARK: - Private Write 
     
     private static func write(validJSON json: Any, toFilename: String) {
-        
         guard JSONSerialization.isValidJSONObject(json) else {
             print(#function, "Invalid JSON")
+            assertionFailure("Invalid JSON! Cannot write to file.")
```

Comment:
The assertion will write its message to Beth console as well. So it probably makes more sense to put that above the print statement.